plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/FileSystemExchangeModule.java

```diff
@@ -68,7 +68,7 @@ else if (ImmutableSet.of("abfs", "abfss").contains(scheme)) {
             configBinder(binder).bindConfig(ExchangeAzureConfig.class);
         }
         else {
-            throw new TrinoException(NOT_SUPPORTED, format("Scheme %s is not supported as exchange spooling storage", scheme));
+            binder.addError(new TrinoException(NOT_SUPPORTED, format("Scheme %s is not supported as exchange spooling storage", scheme)));
```

Comment:
what does it change really? Isn't exception caught up the stack anyway and handle same way?
We are throwing in line 52 and this one is no different, right?

Reply:
But still:
what does it change really?

Reply:
Actually yeah - that way we may collect more than one error.

Reply:
I tested and just realized that it doesn't make much of a difference in terms of error surfacing. Added return under line 52. I'm fine both ways.